src/ode_solve.jl

```diff
@@ -330,10 +331,24 @@ function generate_loss(strategy::WeightedIntervalTraining, phi, f, autodiff::Boo
     return loss
 end
 
+function generate_additional_points_loss(tstops, phi, f, autodiff::Bool, tspan, p, batch)
```

Comment:
Also this needs a better name. `additional_loss` is something different, and this is about `tstops`, not `additional_loss`, so call it `evaluate_tstops_loss`

Reply:
Fixed